### PR TITLE
Feature/462923 create filename db latest

### PR DESCRIPTION
**Pull request by Skumarhpa**

### CODE REVIEW

File: src/EPR.Calculator.API.UnitTests/CreateDefaultParameterDataValidatorTest.cs [hunk 1]
```diff
@@ -114,7 +114,8 @@ public void ValidateTest_For_Missing_Unique_References()
             var dto = new CreateDefaultParameterSettingDto
             {
                 ParameterYear = "2024-25",
```
Comment (udaydefra) [inline src/EPR.Calculator.API.UnitTests/CreateDefaultParameterDataValidatorTest.cs:116]: Do you want to add a Unit test for 
No File name being sent
Max Size 
Empty String

**Review by johnbenjamin-defra — DISMISSED** [review]

File: src/EPR.Calculator.API.UnitTests/CreateDefaultParameterDataValidatorTest.cs (hunk 1 above)
Comment (Skumarhpa) [reply src/EPR.Calculator.API.UnitTests/CreateDefaultParameterDataValidatorTest.cs:116]: Resolved comments

File: src/EPR.Calculator.API.UnitTests/DefaultParameterSettingControllerTest.cs [hunk 2]
```diff
@@ -102,6 +102,81 @@ public void GetSchemeParameter_Return_400_Error_With_No_YearSupplied()
...
+                        ParameterUniqueReferenceId = item
+                    });
+
+                }
+            }
+
+
+            CreateDefaultParameterSettingValidator _validator = new CreateDefaultParameterSettingValidator();
+            CreateDefaultParameterSettingDto _parameter = new CreateDefaultParameterSettingDto()
+            { ParameterFileName = "wwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwww", ParameterYear = "2024-25", SchemeParameterTemplateValues = schemeParameterTemplateValues };
```
Comment (udaydefra) [inline src/EPR.Calculator.API.UnitTests/DefaultParameterSettingControllerTest.cs:171]: `var content = new string('A', fileSizeInKB);`

Comment (Skumarhpa) [reply src/EPR.Calculator.API.UnitTests/DefaultParameterSettingControllerTest.cs]: updated as suggested. Thanks.

**Review by udaydefra — APPROVED** [review]

**Review by Mazar-Shaik — APPROVED** [review]

**Review by johnbenjamin-defra — DISMISSED** [review]

**Review by Mazar-Shaik — APPROVED** [review]

**Review by rekhadevieviden — APPROVED** [review]

**Review by udaydefra — APPROVED** [review]

**Review by johnbenjamin-defra — APPROVED** [review]

**Pull request merged**